app/Filament/Resources/GameResource.php

```diff
@@ -0,0 +1,340 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources;
+
+use App\Filament\Extensions\Resources\Resource;
+use App\Filament\Resources\GameResource\Pages;
+use App\Filament\Rules\ExistsInForumTopics;
+use App\Filament\Rules\IsAllowedGuideUrl;
+use App\Models\Game;
+use App\Models\System;
+use App\Models\User;
+use Filament\Forms;
+use Filament\Forms\Form;
+use Filament\Infolists;
+use Filament\Infolists\Infolist;
+use Filament\Pages\Page;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Contracts\Support\Htmlable;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletingScope;
+use Illuminate\Support\Facades\Auth;
+
+class GameResource extends Resource
+{
+    protected static ?string $model = Game::class;
+
+    protected static ?string $navigationIcon = 'fas-gamepad';
+
+    protected static ?string $navigationGroup = 'Platform';
+
+    protected static ?int $navigationSort = 1;
+
+    protected static ?string $recordTitleAttribute = 'title';
+
+    protected static int $globalSearchResultsLimit = 5;
+
+    /**
+     * @param Game $record
+     */
+    public static function getGlobalSearchResultTitle(Model $record): string|Htmlable
+    {
+        return $record->title;
+    }
+
+    /**
+     * @param Game $record
+     */
+    public static function getGlobalSearchResultDetails(Model $record): array
+    {
+        return [
+            'ID' => $record->id,
+            'System' => $record->system->name,
+        ];
+    }
+
+    public static function getGloballySearchableAttributes(): array
+    {
+        return ['ID', 'Title'];
+    }
+
+    public static function infolist(Infolist $infolist): Infolist
+    {
+        return $infolist
+            ->schema([
+                Infolists\Components\ImageEntry::make('badge_url')
+                    ->label('')
+                    ->size(config('media.icon.lg.width')),
+
+                Infolists\Components\Section::make('Metadata')
+                    ->columns(['md' => 2, 'xl' => 3, '2xl' => 4])
+                    ->schema([
+                        Infolists\Components\TextEntry::make('id')
+                            ->label('ID'),
+
+                        Infolists\Components\TextEntry::make('title'),
+
+                        Infolists\Components\TextEntry::make('forumTopic.id')
+                            ->label('Forum Topic')
+                            ->url(fn (?int $state) => url("viewtopic.php?t={$state}")),
+
+                        Infolists\Components\TextEntry::make('system')
+                            ->formatStateUsing(fn (System $state) => "[{$state->id}] {$state->name}")
+                            ->url(fn (System $state) => SystemResource::getUrl('view', ['record' => $state->id])),
```

Comment:
This link gets generated even if I don't have access.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/8e6df0f0-7f65-4171-b910-ac7816e08c72)

Reply:
Good catch - I didn't think of that. Fixed in latest.

Reply:
The link still exists in the View page:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/2b58632f-e79f-4698-9d27-55c365d25b8c)

Reply:
Forgot about that one. Updated in latest.